cyclonedx/model/bom.py

```diff
@@ -37,11 +37,11 @@ class Tool:
         See the CycloneDX Schema for toolType: https://cyclonedx.org/docs/1.3/#type_toolType
     """
 
-    def __init__(self, vendor: str, name: str, version: str, hashes: List[HashType] = []):
```

Comment:
the default `[]` list was mitable.
see https://blog.finxter.com/the-mutable-default-argument-in-python/

Reply:
added `flake8-bugbear` to the project, which would have found such mistakes.